### ISSUE TITLE
darksoftware.net: detection

### ISSUE THREAD

**Issue by toastysteve** [uBlockOrigin/uAssets#16924]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other [similarly-purposed blockers](https://twitter.com/gorhill/status/1033706103782170625).
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://darksoftware.net/forums/switch-nsp-xci-roms.50/`

### Category

detection

### Description

Most of the forum is browsable, but when you open any "ROMS" page, you're greeted with this ^ after a short delay.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://user-images.githubusercontent.com/58093690/221176660-1f9d84b8-4303-44c3-af21-86bda56e53d1.png)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.47.2
Chromium: 110
filterset (summary): 
  network: 83604
  cosmetic: 38735
  scriptlet: 16972
  html: 0
listset (total-discarded, last-updated): 
  added: 
    https://gitcdn.xyz/repo/NanoMeow/MDLMirror/master/hosts.txt: 0-0, 23h.34m
    https://mirror.cedia.org.ec/malwaredomains/justdomains: 0-0, never
  default: 
    user-filters: 7-0, never
    easylist: 62339-47, 2d.2h.44m
    easyprivacy: 30963-1172, 2d.2h.45m
    plowe-0: 3675-906, 5d.2h.36m
    ublock-abuse: 77-0, 8d.23h.20m
    ublock-badware: 4432-122, 3d.54m
    ublock-filters: 32027-141, 3d.52m
    ublock-privacy: 273-1, 10d.28m
    ublock-quick-fixes: 322-18, 3h.9m
    ublock-unbreak: 1904-44, 3d.53m
    urlhaus-1: 5916-0, 3h.10m
filterset (user): [array of 7 redacted]
trustedset: 
  added: [array of 14 redacted]
  removed: 
    edge-scheme
switchRuleset: 
  added: [array of 3 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 3535 ms (selfie)
  maxAssetCacheWait: 104 ms
popupPanel: 
  blocked: 2
  blockedDetails: 
    add.pics: 1
    googletagmanager.com: 1
```
</details>

**Comment by mapx-** [comment]
I get no detection on ROMS pages.

**Comment by garry-ut99** [comment]
I don't have detection as a guest, but the page says: "You must be logged-in to do that. " and I don't have an account, so I'm unable to verify what happens after logging-in.

**Comment by iam-py-test** [comment]
> ```yml
>    https://gitcdn.xyz/repo/NanoMeow/MDLMirror/master/hosts.txt: 0-0, 23h.34m
>    https://mirror.cedia.org.ec/malwaredomains/justdomains: 0-0, never
>```

The first list is dead (404) and the second one hasn't been updated in years. Try removing these & updating your filters.
However, I can reproduce (sent details privately)

**Comment by stephenhawk8054** [comment]
I don't see any anti-adb on the link even after signing in: https://user-images.githubusercontent.com/66517106/221332945-0cd13841-4346-408a-8604-342019eff242.png ?

**Comment by mapx-** [comment]
@toastysteve 

- clear cookies / localStorage/ cache for the tested page
- reset uBO, don't modify any uBO setting, update all the filter lists, don't add any other list (other than your regional list)
- disable any custom filter for that site
- disable any other extension / addon in your browser, disable builtin blockers /shields/ tracking protection in brave/ opera/ vivaldi/ edge/ firefox etc
- if Opera browser: open opera://extensions/ and check the "allow access to search page results" option
- disable (if any) the web shields in your antivirus / antimalware software / other blocking software
- are you using a DNS which features adblocking (like AdGuard DNS)?=> test using another DNS (google for example =>  IP addresses 8.8.8.8 & 8.8.4.4)
- restart the system, reopen the browser, test again.

**Comment by toastysteve** [comment]
Apologies all. I should have done more testing and provided more information. 
Using Ublock in Firefox, I found I had no issues with detection.
It's detecting an adblocker when using Edge (my primary browser).
I do use a Pi-Hole and route DNS through that, but I'd tried whitelisting darksoftware.net in there, and the fact that FF is working fine disqualified that as the cause.
Same issue occured when using incognito/in private browsing in Edge.
In the end I disabled uBlock for the domain, cleared the cache for that domain, and then I had to close down the browser (refreshing didn't work). Once back in, ads started displaying again and I was able to re-enable uBlock without detection.

Thanks for the help and guidance it's very much appreciated. Keep up the good work.
